templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -594,3 +594,23 @@ raise UnexpectedWebhookEventType(webhook_name, event_type)
 
 `webhook_name` is the name of the integration that raises the exception.
 `event_type` is the name of the unexpected webhook event.
+
+
+### Working with a locally hosted third-party service
+
+Some third-party services like Jira and Bitbucket Server can be hosted locally
+and it's possible to allow these services to be able to directly post to your
+Zulip development enviroment. But first you'll need to make sure that your Zulip
+development environment (notably the virtual machine) is visible to the locally
+hosted service. Here's how you can do that:
+
+1. Open zulip/Vagrantfile. Replace the line `host_ip_addr = "127.0.0.1"` with
```

Comment:
Does it make sense to link to the Vagrantfile here?
```suggestion
1. Open [zulip/Vagrantfile](https://github.com/zulip/zulip/blob/75635844dbaaefede1938f6372b0baa09d93b4b7/Vagrantfile#L54). Replace the line `host_ip_addr = "127.0.0.1"` with
```

Reply:
Well, IMHO, I think that it's kind of unnecessary since in the end, the user/developer would have to ultimately modify their local copy of the file. Thus they would have to open the file locally anyways.